src/routes/game/GameView.vue

```diff
@@ -750,7 +750,7 @@ export default {
       });
     },
     topCard(newTopCard, oldTopCard) {
-      if (oldTopCard && !newTopCard) {
+      if (this.gameStore.id && oldTopCard && !newTopCard) {
```

Comment:
This is the fix. The snackbar would trigger during the rematch flow because there was a moment where the gameStore would be reset so the topCard was null and it was previously not null (during the last game), but at that moement the game's id is also set to null. So this check ensures that we only show the snackbar duing an actual game